brands/public_transport/station.json

```diff
@@ -0,0 +1,16 @@
+{
+  "public_transport/station|Greyhound Station": {
+    "countryCodes": ["us"],
+    "matchNames": ["greyhound terminal"],
+    "tags": {
+      "amenity": "bus_station",
+      "brand": "Greyhound Lines",
+      "brand:wikidata": "Q755309",
```

Comment:
OSM uses `network` and `operator` tags for public transportation facilities, both of which come with `:wikidata` subkeys as well. I suppose `brand` would indicate a relationship such as Ford sponsoring the GoBike program in the San Francisco Bay Area, but that’s much less important than capturing the network and operator.